src/L1YearnEscrow.sol

```diff
@@ -281,17 +297,34 @@ contract L1YearnEscrow is L1Escrow {
 
     /**
      * @dev Update the minimum buffer to keep in the escrow.
+     *      uint128 max would be the max buffer.
      * @param _minimumBuffer The new minimum buffer to enforce.
      */
     function updateMinimumBuffer(
         uint256 _minimumBuffer
     ) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
         VaultStorage storage $ = _getVaultStorage();
-        $.minimumBuffer = _minimumBuffer;
+        require(_minimumBuffer <= type(uint128).max, "max size");
```

Comment:
why not change the parameter's type to uint128?

Reply:
I tend to prefer keeping the external types the generic versions to make integrations and interfaces easier.

But can change it to 128 to make is clearer

Reply:
https://github.com/yearn/yearn-stb/pull/17/commits/db539770778790eb87070299e02103f0c3c9af82